src/registrar/templates/portfolio_base.html

```diff
@@ -4,7 +4,7 @@
   <div id="wrapper" class="{% block wrapper_class %}dashboard--portfolio{% endblock %}">
     {% block content %}
         
-        <main class="grid-container">
+        <main id="main-content" class="grid-container {% if is_widescreen_mode %} grid-container--widescreen {% endif %}">
```

Comment:
BLOCKING: Are we positive this is not a duplicate? I think the child templates all set the main-content id, so this is a dupe id.

Context: the reason main-content is left to the child templates to set up is because the org page has a side nav and we want the skip-navigation to point to the content AFTER the sidenav.

Reply:
Grr.. I thought I removed this in a merge (there was a merge conflict).  This wasn't supposed to be added.  Thanks for the catch